### ISSUE TITLE
Making a Highchart object

### ISSUE THREAD

**Issue by mistercrunch** [gtnx/pandas-highcharts#22]
I was thinking about refactoring your code a little bit by making a `Highchart` object as in `class Highchart(object):` and would make the `serialize` function use this object and behave the exact same way it does at the moment. 

I want to do that to be able to generate the object (without using the `serialize` function), alter the data structure at will, and then render it my way.

Would you review / accept such a PR?

**Comment by gtnx** [comment]
Hello,

Sorry for the late answer. If i understood correctly, you want an `Highchart` class so that you can inherit from it and implement your own rendering?
If yes it would be nice

**Comment by mistercrunch** [comment]
I ended up building my own class and reused some of your code along the way:
https://github.com/mistercrunch/panoramix/blob/master/app/highchart.py

I realized that in my context I didn't really have a need to maintain the `pandas.plot()` compatibility so I moved away from it as I understand its a design decision in `pandas_highcharts` and a bit of a constraint for me at this point. 

It was really nice to use `pandas_highcharts` to get my project going though, it took no time to get great results. I'm just growing my project to do more custom things at this point.

Thanks!
